news/2024/2024-10-09-resurrection-cup-2024-results.md

```diff
@@ -0,0 +1,417 @@
+
+Her personality is a bit complex, she is individualistic, bit egoistic, always the center of attention, but still very much poised. But if you peel back the tough shield, she is really sweet, pure, angelic, and a best friend you can always count on, someone that will fight for you no matter what.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/12840110" style="background-image: url('/wiki/shared/news/2024-10-09-resurrection-cup-2024-results/avatar-Phreel.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/12840110">Phreel</a></p>
+
+Were there any challenges?
+
+<a class="avatar news-chat-quote__avatar" href="https://twitter.com/Yumeyo_art" style="background-image: url('/wiki/shared/news/2024-10-09-resurrection-cup-2024-results/avatar-Yumeyo.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://twitter.com/Yumeyo_art">Yumeyo</a></p>
+
+Trying to communicate her personality through art was a tough task, as I have made OCs before, but this was my first experience making a character that I knew will be in front of many people, and also representing Resurrection Cup, and the community working behind it.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/12840110" style="background-image: url('/wiki/shared/news/2024-10-09-resurrection-cup-2024-results/avatar-Phreel.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/12840110">Phreel</a></p>
+
+What does Serenity represent about Resurrection Cup?
+
+<a class="avatar news-chat-quote__avatar" href="https://twitter.com/Yumeyo_art" style="background-image: url('/wiki/shared/news/2024-10-09-resurrection-cup-2024-results/avatar-Yumeyo.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://twitter.com/Yumeyo_art">Yumeyo</a></p>
+
+For me, she is meant to represent how individuals come together to create something amazing for the community. So many talented artists come together to guarantee the best experience for attendees, and Serenity is meant to represent the individualism that with time and friendship forges into an amazing team-effort, creating a creative space for everyone while working towards the same goal. Also in my opinion, a mascot for a tournament with so many custom songs is a really useful thing, as it makes a great starting point for custom map backgrounds, jacket art etc.
+
+</div>
+
+## Staff interviews
+
+A lot of people were involved in making Resurrection Cup happen. We've asked a few more of those people to give insights into what makes Resurrection Cup... Resurrection Cup.
+
+::{#alvearia-and-luminous-sky}::
+
+![](/wiki/shared/news/2024-10-09-resurrection-cup-2024-results/alvearia-and-luminous-sky.jpg)
```

Comment:
these interviews are missing introductory paragraphs ([example](https://osu.ppy.sh/home/news/2024-10-07-osumania-4k-world-cup-2024-concludes#interviews)), since most of them require context to be understood

Reply:
All interviewees now have introductions. Also added horizontal lines to better sepparate the different interviews.